### PR TITLE
feat: #606 add exchangekey list page

### PR DESCRIPTION
**Pull request by Aria622**
Add exchangekey list page tsx and css file.

### CODE REVIEW

File: src/app/addmarket/page.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import { Layout, Button, message } from 'antd';
+import axios from 'axios';
+import RootLayout from '../layout';
+import { useNavigate } from 'react-router-dom'; // 使用 useNavigate 钩子
```
Comment (Yingxue0323) [inline src/app/addmarket/page.tsx:7]: remove Chinese

File: src/app/addmarket/page.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            </div>
+            <div className="field">
+              <label className="label">Select Exchange</label>
+              <div className="control">
+                <ul className="choice is-clearfix">
+                  {exchangeList.map((item) => (
+                    <li
+                      key={item.exchange}
+                      className={`exchange-item ${formData.exchange === item.exchange ? 'active' : ''}`}
+                      // onClick={() => handleSelectExchange(item.exchange)}
```
Comment (Yingxue0323) [inline src/app/addmarket/page.tsx:76]: why not using this onClick function?

Comment (Aria622) [reply src/app/addmarket/page.tsx:76]: Because it depends on other unfinished parts, otherwise it will report an error.

**Review by ucfyao — APPROVED** [review]
done

**Pull request merged**